src/alpine/.devcontainer/devcontainer.json

```
@@ -1,16 +1,19 @@
 {
 	"name": "Alpine",
-	"image": "mcr.microsoft.com/devcontainers/base:alpine-${templateOption:imageVariant}",
+	"image": "mcr.microsoft.com/devcontainers/base:alpine-${templateOption:imageVariant}"
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {}
 
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
 	// "postCreateCommand": "uname -a",
 
-	// Replace when using a ptrace-based debugger like C++, Go, and Rust
-	// "runArgs": [ "--init", "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+	// Configure tool-specific properties.
+	// "customizations": {}
 
-	// Set `remoteUser` to `root` to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-	"remoteUser": "vscode"
+	// Comment out to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```

Should this be "Uncomment?"
```suggestion
	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```

---

Oops! changed.